### ISSUE TITLE
Git Changes happened when work on both of Mac And Windows

### ISSUE THREAD

**Issue by justbilt** [AppLovin/AppLovin-MAX-Unity-Plugin#174]
![image](https://user-images.githubusercontent.com/3746991/174462731-9d1fdd46-e77e-428a-8f8f-13e54eecb747.png)

All of my git working copy are fill with useless change such as this.

**Comment by justbilt** [comment]
https://github.com/AppLovin/AppLovin-MAX-Unity-Plugin/blob/bad1684b8a9f4ae4ca42c24cbddb51a546bffe8c/DemoApp/Assets/MaxSdk/Scripts/IntegrationManager/Editor/AppLovinIntegrationManager.cs#L643

I think change this line to below could fix this issue:
```
            var exportPathLabel = "al_max_export_path-" + assetPath.Replace(pluginParentDir, "").Replace(Path.DirectorySeparatorChar, Path.AltDirectorySeparatorChar);;
```

**Comment by jacattrongnlh** [comment]
I agree that this issue need to be fixed. There are too many changes spawned when different developers on different platform work together

**Comment by santoshbagadi** [comment]
We have released a fix for this issue in [v5.5.3](https://github.com/AppLovin/AppLovin-MAX-Unity-Plugin/releases/tag/release_5_5_3) of our plugin.
